Objective-C/CBLReplicator.h

```diff
@@ -160,7 +160,7 @@ typedef struct {
  @param error error On return, the error if any.
  @return A  set of document Ids, each of which has one or more pending revisions
  */
-- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error;
+- (NSSet<NSString*>*) pendingDocumentIds: (NSError**)error __attribute__((swift_error(nonnull_error)));
```

Comment:
i couldn't find any Apple documentation, but clang doc has [this](https://github.com/apple/swift-clang/blob/383859a9c4b964af3d127b5cc8abd0a8f11dd164/include/clang/Basic/AttrDocs.td#L1800-L1819), also can seen in FBSDK